src/General/html/Features/QuickReply.html

```diff
@@ -36,6 +36,7 @@
       <input type=checkbox id=qr-file-spoiler title='Spoiler image'>
     </label>
     <input type=submit>
+    #{if parseInt(Conf['additionalCooldown']) > 0 then '<span><input type=checkbox data-name=addCooldown title="Add Custom Cooldown">Enable Custom Cooldown</span>' else ''}
```

Comment:
Here the `<span>` would be better as a `<label>`, and I would get rid of the `if`, give it an ID, and hide it with CSS:

    #file-n-submit:not(.add-cooldown) #add-cooldown-label {
      display: none;
    }